share/sv.po

```diff
@@ -2,75 +2,76 @@ msgid ""
 msgstr ""
 "Project-Id-Version: 1.0.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-11-17 08:57+0100\n"
-"PO-Revision-Date: 2021-11-17 17:32+0100\n"
+"POT-Creation-Date: 2022-05-22 15:34+0000\n"
+"PO-Revision-Date: 2022-05-22 15:23+0000\n"
 "Last-Translator: mats.dufberg@iis.se\n"
 "Language-Team: Zonemaster project\n"
 "Language: sv\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-msgid "Domain name required"
-msgstr "Domännamn är obligatoriskt"
+msgid "Invalid method parameter(s)."
+msgstr "Ogiltig metodparameter."
+
+msgid "Missing property"
+msgstr "Attribut saknas"
 
 msgid ""
-"The domain name is not a valid IDNA string and cannot be converted to an A-"
-"label"
+"Warning: Zonemaster::LDNS not compiled with IDN support, cannot handle non-"
+"ASCII names correctly."
 msgstr ""
-"Domännamnet är inte ett giltigt IDN-namn och kan inte konverteras till A-"
-"label-format"
+"Varning: Zonemaster::LDNS är inte kompilerad med IDNA-stöd, så enbart ASCII-"
+"namn kan hanteras."
+
+msgid "Invalid digest format"
+msgstr "Ogiltigt format på DS-digest"
```

Comment:
Have you considered removing the reference to DS from the msgstr? The msgid speaks of digests in general and does not mention the DS record type. This means that an identical msgid could be used in another place where the DS reference isn't applicable. That would render the Swedish translation in that place incorrect, and we would have no good way of noticing.